backends-velox/src/main/scala/io/glutenproject/backendsapi/velox/MetricsApiImpl.scala

```diff
@@ -37,6 +37,20 @@ class MetricsApiImpl extends MetricsApi with Logging {
     MetricsUtil.updateNativeMetrics(child, relMap, joinParamsMap, aggParamsMap)
   }
 
+  override def genInputIteratorTransformerMetrics(
+      sparkContext: SparkContext): Map[String, SQLMetric] = {
+    Map(
+      "cpuCount" -> SQLMetrics.createMetric(sparkContext, "cpu wall time count"),
+      "wallNanos" -> SQLMetrics.createNanoTimingMetric(sparkContext, "totaltime of batch scan"),
```

Comment:
Is it accurate to use "totaltime of batch scan"? I thought the transformer is common for both scan and other inputs.

Reply:
good catch, I think it is copied from a wrong code path..